packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1120,3 +1133,26 @@ export async function pushDataToQueue(data: UploadType, requestQueueHandler: Req
 }
 
 export const sleep = (ms = 100) => new Promise((resolve) => setTimeout(resolve, ms))
+
+export async function uploadLogs(user: string | undefined, key: string | undefined, clientBuildUuid: string) {
+    const fileStream = fs.createReadStream(BStackLogger.logFilePath)
+    const uploadAddress = UPLOAD_LOGS_ADDRESS
+    const zip = zlib.createGzip({ level: 1 })
+    fileStream.pipe(zip)
+
+    const formData : any = new FormData()
```

Comment:
Let's avoid usage of `any` as much as possible

Reply:
Oops, Thanks for pointing out. Will try this.